pages/community/events/index.js

```diff
@@ -0,0 +1,117 @@
+import GenericLayout from '../../../components/layout/GenericLayout';
+
+function index() {
+      const image = '/img/social/website-card.png';
+  let {navItems} = useContext(EventsContext)
+  const [events, setEvents] = useState(navItems)
+  return (
+    <GenericLayout
+      title="AsyncAPI events"
+      description="Our catalogs of events and meetups"
+      image={image}
+      hideBanner={true}
+      wide
+    >
+      <div className="mt-10 sm:mt-28">
+        <div className="w-full sm:w-9/12">
+          <h1 className="font-semibold text-3xl lg:text-7xl leading-tight md:text-4xl">
+            Join a AsyncAPI event, from anywhere in the world.
+          </h1>
+        </div>
+        <div className="flex justify-end mt-10">
+          <div className="w-[80%] sm:w-1/3">
+            <Heading
+              level="h2"
+              typeStyle="body-lg"
+              textColor="text-gray-700"
+              className="text-slate-500"
+            >
+              "Learn more about our live and recorded events below. You can also
+              sign up to our community newsletter to stay up-to-date on our
+              events."
+            </Heading>
+          </div>
+        </div>
+        <hr className="mt-20 border-dotted border-t-2 border-black " />
+      </div>
+      <div className="mt-24 sm:flex justify-between items-center h-auto sm:h-[400px]">
+        <div className="w-full sm:w-[55%] h-full bg-confBg rounded-md bg-cover bg-center p-10 flex flex-col justify-between text-white">
+          <img
+            src="/img/logos/conflogo.png"
+            alt="conf-logo"
+            className="w-[100px] sm:w-[150px]"
+          />
+          <div className="w-full sm:w-[70%]">
+            <Heading
+              level="h2"
+              typeStyle="heading-sm-semibold"
+              className="mt-10"
+            >
+              Join us at the AsyncAPI 2022 conference with thousands around the
+              world for free
+            </Heading>
+            <a
+              href="https://conference.asyncapi.com/"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <div className="mt-5 flex items-center">
+                <span>Join us now </span>
+                <ArrowRightIcon className="w-[20px] ml-3 mt-1" />
+              </div>
+            </a>
+          </div>
+        </div>
+        <div className="h-full mt-10 sm:mt-0 w-full sm:w-[43%] bg-secondary-600 rounded-md flex">
+          <div className="w-[50%] h-full hidden sm:flex sm:flex-col">
+            <div className="h-[50%] bg-officeHourCover bg-center bg-cover rounded-tl-md"></div>
+            <div className="h-[50%]  bg-patternCover bg-center bg-cover rounded-bl-md" />
+          </div>
+          <div className="text-white w-full sm:w-[50%] p-5 flex flex-col items-center justify-center h-full">
+            <div>
+              <Heading level="h2" typeStyle="heading-md-semibold">
```

Comment:
Why is TSC plural? As far as I know, we only have one TSC group, with many members? 🤔